src/observable-hash.js

```diff
@@ -42,14 +42,18 @@ Class.extend(ObservableHash, {
    */
   observersStart: function() {
     this._observers.enabled = true;
-    this._observers.forEach(function(observer) {
-      observer.bind(this._context);
-    }, this);
+    this._observers.forEach(this.observersStartBindHelper.bind(this), this);
```

Comment:
The `bind` is unnecessary when using the second parameter, `context`. It is redundant and will be faster to just use the context argument.